xml/System.Xml.Linq/XElement.xml

```diff
@@ -45,7 +45,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Represents an XML element.  See [XElement Class Overview](http://msdn.microsoft.com/library/d35180fe-7016-4895-9bfc-ba1e3f7875ec) and the Remarks section on this page for usage information and examples.</summary>
+    <summary>Represents an XML element.  See <see href="https://msdn.microsoft.com/library/d35180fe-7016-4895-9bfc-ba1e3f7875ec">XElement Class Overview</see> and the Remarks section on this page for usage information and examples.</summary>
```

Comment:
A better link is to >https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/xelement-class-overview>